src/styles/utils/icon.scss

```diff
@@ -0,0 +1,36 @@
+.icon {
```

Comment:
please, move all BEM-block files from the utils folder to the `blocks` or `components`, etc folder